Sources/MapboxNavigation/RouteVoiceController.swift

```diff
@@ -62,8 +62,12 @@ open class RouteVoiceController: NSObject, AVSpeechSynthesizerDelegate {
      */
     public init(navigationService: NavigationService, speechSynthesizer: SpeechSynthesizing? = nil, accessToken: String? = nil, host: String? = nil) {
         self.speechSynthesizer = speechSynthesizer ?? MultiplexedSpeechSynthesizer(accessToken: accessToken, host: host)
+        rerouteSoundPlayer = try! AVAudioPlayer(data: NSDataAsset(name: "reroute-sound", bundle: .mapboxNavigation)!.data,
```

Comment:
Why does it double the existing initialization below?

Reply:
It's not needed actually, I'd like to have it in initializer only. Updated code.